optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-api/pom.xml

```diff
@@ -32,4 +32,25 @@
   <name>OptaPlanner Workbench - Domain Editor API</name>
   <description>OptaPlanner Workbench - Domain API</description>
 
+
+  <dependencies>
+
+    <!-- dependencies added because of new illegal transitive dependency check -->
```

Comment:
Just to explain this a bit more (I know this comment is all over the poms, nut just here): the dependencies were not added because of the "illegal transitive dependency check", but because its a good Maven practice to declare all direct dependencies. The "illegal transitive dependency check" is just a mean to detect these violations. No need to fix it now though, we should do that in all repos via find&replace all at once.